src/com/dynatrace/loadrunner/logic/Constants.java

```diff
@@ -1,31 +1,37 @@
 package com.dynatrace.loadrunner.logic;
 
 public class Constants {
-	public final static String ADD_HEADER_FUNCTION="addDynaTraceHeader";
+	public final static String ADD_HEADER_FUNCTION_TEST="addDynatraceHeaderTest";
 
 	public static final String CRLF_OS_INDEPENDENT = "\r\n";
+	
+	public final static String DT_FUNCTION_TEST = CRLF_OS_INDEPENDENT + "void addDynatraceHeaderTest(char* header){" + CRLF_OS_INDEPENDENT
```

Comment:
definitely needs to be moved to a separate file!
needs checking if CRLF behaves correctly then on different OSs